programs/compressed-pda/src/sdk/invoke.rs

```diff
@@ -219,13 +214,13 @@ pub fn create_execute_compressed_instruction(
 
 //     let mut inputs = Vec::new();
 
-//     let accounts = crate::accounts::TransferInstruction {
+//     let accounts = crate::accounts::InvokeInstruction {
```

Comment:
can these be removed?

Reply:
let's remove this when we merge the optimized compressed pda instruction data